src/main/java/com/sparta/vicky/user/entity/User.java

```diff
@@ -0,0 +1,54 @@
+package com.sparta.vicky.user.entity;
+
+import com.sparta.vicky.baseEntity.Timestamped;
+import com.sparta.vicky.user.dto.SignupRequestDto;
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.ColumnDefault;
+
+import java.time.LocalDateTime;
+
+@Getter
+@Setter
+@Table(name = "user")
+public class User extends Timestamped {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(unique = true)
+    private String username;
+
+    @Column(nullable = false)
+    private String password;
+
+    @Column(nullable = false)
+    private String name;
+
+    @Column
+    private String email;
+
+    @Column
+    private String introduce;
+
+    @Column
+    @ColumnDefault("'정상'")
+    private String statusCode;
+
+    @Column
+    private String refreshToken;
+
+    @Column(nullable = false)
+    private LocalDateTime statusUpdatedAt;
+
+    public User(SignupRequestDto requestDto) {
+        this.username = requestDto.getUsername();
+        this.password = requestDto.getPassword();
+        this.name = requestDto.getName();
+        this.email = requestDto.getEmail();
+        this.introduce = requestDto.getIntroduce();
+    }
+
```

Comment:
엔티티에 기본 생성자가 있어야 해서 추가 부탁드려요!
```suggestion
    protected User() {}
```

Reply:
아 그리고 @Entity 애노테이션도 없네요!